app/controllers/concerns/param_converters.rb

```diff
@@ -15,4 +15,41 @@ def false?(value)
   def nil_param?(value)
     value == 'null' ? nil : value
   end
+
```

Comment:
that's what I meant by creating a class that can handle these search params ... it's not very clean code if those skill search related param helpers are just in this generic concern/module.

so it would be nice to have this in a own class ... something like PeopleSkills::FilterParams class I suggested before.

to me it also looks like there's to much manual work for converting the params ... rails params are pretty standard and I'm sure there's helpers to handle these conversions without to much implementation/code